gallery/src/pageConfig.ts

```diff
@@ -140,141 +140,151 @@ import NxStatefulCollapsibleRadioSelectPage
+    'Stateful Icon Dropdown': { content: NxStatefulIconDropdownPage, type: 'react' },
+    'Pagination': { content: NxPaginationPage, type: 'react' },
+    'Indeterminate Pagination': { content: NxIndeterminatePaginationPage, type: 'react' },
+    'Search Dropdown': { content: NxSearchDropdownPage, type: 'react' },
+    'Stateful Search Dropdown': { content: NxStatefulSearchDropdownPage, type: 'react' },
+    'Text Link': { content: NxTextLinkPage, type: 'react' },
+    'Tooltip': { content: NxTooltipPage, type: 'react' },
+    'Overflow Tooltip': { content: NxOverflowTooltipPage, type: 'react' }
+  },
+  'Forms': {
+    'Checkbox': { content: NxCheckboxPage, type: 'react' },
+    'Stateful Checkbox': { content: NxStatefulCheckboxPage, type: 'react' },
+    'Code Snippet': { content: NxCodeSnippetPage, type: 'react' },
+    'Collapsible MultiSelect': { content: NxCollapsibleMultiSelectPage, type: 'react' },
+    'Stateful Collapsible MultiSelect': { content: NxStatefulCollapsibleMultiSelectPage, type: 'react' },
+    'CollapsibleRadioSelect': { content: NxCollapsibleRadioSelectPage, type: 'react' },
+    'Stateful Collapsible Radio Select': { content: NxStatefulCollapsibleRadioSelectPage, type: 'react' },
+    'Color Picker': { content: NxColorPickerPage, type: 'react' },
+    'Date Input': { content: NxDateInputPage, type: 'react' },
+    'Stateful Date Input': { content: NxStatefulDateInputPage, type: 'react' },
+    'Fieldset': { content: NxFieldsetPage, type: 'react' },
+    'Filter Input': { content: NxFilterInputPage, type: 'react' },
+    'Form': { content: NxFormPage, type: 'react' },
+    'Form Group': { content: NxFormGroupPage, type: 'react' },
+    'Form Select': { content: NxFormSelectComponentPage, type: 'react' },
+    'Policy Threat Slider': { content: NxPolicyThreatSliderPage, type: 'react' },
+    'Radio': { content: NxRadioPage, type: 'react' },
+    'Search Transfer List': { content: NxSearchTransferListPage, type: 'react' },
+    'Stateful Search Transfer List': { content: NxStatefulSearchTransferListPage, type: 'react' },
+    'Text Input': { content: NxTextInputPage, type: 'react' },
+    'Stateful Text Input': { content: NxStatefulTextInputPage, type: 'react' },
+    'Toggle': { content: NxTogglePage, type: 'react' },
+    'Stateful Toggle': { content: NxStatefulTogglePage, type: 'react' },
+    'Transfer List': { content: NxTransferListPage, type: 'react' },
+    'Stateful Transfer List': { content: NxStatefulTransferListPage, type: 'react' },
+    'Tree View Multi-Select': { content: NxTreeViewMultiSelect, type: 'react' },
+    'Stateful Tree View Multi-Select': { content: NxStatefulTreeViewMultiSelect, type: 'react' },
+    'Tree View Radio Select': { content: NxTreeViewRadioSelectPage, type: 'react' },
+    'Stateful Tree View Radio Select': { content: NxStatefulTreeViewRadioSelectPage, type: 'react' }
+  },
+  'Data Presentation': {
+    'Accordion': { content: NxAccordionPage, type: 'react' },
```

Comment:
Thoughts on Accordion being here vs in Layout.  I would've expected it to be in the same bin as tile and modal

Reply:
> Thoughts on Accordion being here vs in Layout? I would've expected it to be in the same bin as tile and modal

I can see it in either place myself. It's kind of like a definition list and a tile. Both have "titles" both have "contents". I don't have strong feelings.

Reply:
> I can see it in either place myself. It's kind of like a definition list and a tile. Both have "titles" both have "contents". I don't have strong feelings.

Lists don't really have titles though, they're just shown next to titles in the examples